server/routes/location.js

```diff
@@ -39,17 +46,10 @@ module.exports = {
     return h.view('location', { model })
   },
   options: {
-    validate: {
-      query: joi.object({
-        q: joi.string().trim().max(200).required(),
-        cz: joi.string(),
-        l: joi.string(),
-        btn: joi.string(),
-        ext: joi.string(),
-        fid: joi.string(),
-        lyr: joi.string(),
-        v: joi.string()
-      })
-    }
+    // validate: {
```

Comment:
@JFSCH I think this code needs reinstating, it is to stop a massive string being input which causes an unhandled route balancing 500 error

Reply:
This error: 201111/081954.104, [log,connection,client,error] message: Parse Error: Header overflow, stack: Error: Parse Error: Header overflow
 if you paste a really long string into the query param

Reply:
Code reinstated as suggested